templates/flavors/etcd-backup-restore/kustomization.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
```

Comment:
instead of pulling in `../base` and these other files could we just pull in `../default`?

Reply:
done!